admin/onboarding/onboard.md

```diff
@@ -1,10 +1,19 @@
 # iNeedA\<br> Onboarding Guide
-Updated 6/3/21
+Updated 6/6/21
 
 ## Navigation
 - [iNeedA\<br> Onboarding Guide](#ineedabr-onboarding-guide)
```

Comment:
We should add a guide of how we expect things to be contributed to the repo (what is the main PR process, where questions could be asked, etc)

Reply:
addressed